infrastructure/snyk_api/snyk_api.go

```diff
@@ -78,28 +91,38 @@ func NewSnykApiClient(client func() *http.Client) SnykApiClient {
 
 func (s *SnykApiClientImpl) SastSettings() (SastResponse, error) {
 	method := "SastSettings"
+	var response SastResponse
 	log.Debug().Str("method", method).Msg("API: Getting SastEnabled")
 	path := "/cli-config/settings/sast"
 	organization := config.CurrentConfig().Organization()
 	if organization != "" {
 		path += "?org=" + url.QueryEscape(organization)
 	}
-	responseBody, err := s.doCall("GET", path, nil)
+
+	err := s.processApiResponse(method, path, &response)
 	if err != nil {
-		fmtErr := fmt.Errorf("%v: %v", err, responseBody)
-		log.Err(fmtErr).Str("method", method).Msg("error when calling sastEnabled endpoint")
+		log.Err(err).Str("method", method).Msg("error when calling sastEnabled endpoint")
 		return SastResponse{}, err
 	}
+	return response, err
+}
 
-	var response SastResponse
-	unmarshalErr := json.Unmarshal(responseBody, &response)
-	if unmarshalErr != nil {
-		fmtErr := fmt.Errorf("%v: %v", err, responseBody)
-		log.Err(fmtErr).Str("method", method).Msg("couldn't unmarshal SastResponse")
-		return SastResponse{}, err
+func (s *SnykApiClientImpl) FeatureFlagSettings(featureFlagType FeatureFlagType) (FFResponse, error) {
+	method := "FeatureFlagSettings"
+	var response FFResponse
+	log.Debug().Str("method", method).Msgf("API: Getting %s", featureFlagType.String())
+	path := path.Join("/cli-config/feature-flag/", string(featureFlagType))
+	organization := config.CurrentConfig().Organization()
+	if organization != "" {
+		path += "?org=" + url.QueryEscape(organization)
+	}
```

Comment:
I'm not sure, this parameter carries over to the feature flag endpoint, or how to specify the org here. It may be, that it uses the default org, we should confirm the behaviour.

Reply:
I've added the test `"Get feature flag status when disabled for a ORG"` to document the behaviour of the API when called with an `org_id` which does not have the FF enabled.

For example, my default org does have `useWorkspaces` enabled but the org `hammerhead` doesn't and it gets the following message:

```json
{
  "ok": false,
  "userMessage": "Org hammerhead doesn't have 'use-workspaces' feature enabled"
}
```

This response can be retrieved by calling: `/v1/cli-config/feature-flags/useWorkspaces?org=org_id`